docs/vmanomaly.md

```diff
@@ -94,6 +94,9 @@ Currently, vmanomaly ships with a few common models:
    A simple moving window of quantiles. Easy to use, easy to understand, but not as powerful as 
    other models.
 
+1. **Isolation Forest**
+   Detects anomalies in *multivariate data* using binary trees. The algorithm has a linear time complexity and a low memory requirement, which works well with high-volume data. See [scikit-learn.org documentation](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.IsolationForest.html) for Isolation Forest.
+
```

Comment:
we can implicitly add here that it can be used on both univatiate and multivariate data. The emphasis should remain on multivariate case. We can also add a hyperlink to warn user about [the curse of dimensionality](https://en.wikipedia.org/wiki/Curse_of_dimensionality) (i.e. "don't use single multivariate model if you expect your queries to return many (M) metrics of shorter length (N), where M >> N) when it's hard for a model to learn meaningful dependencies from too sparse data hypercube